### PR TITLE
Relax Rails dependency to < 6.1

### PR DESCRIPTION
**Pull request by glebm**
Rails 6.0 is about to be released

### CODE REVIEW

**Comment by Mange** [comment]
I'd like you to embed a Rails 6 application for the integration tests.

**Comment by glebm** [comment]
@Mange Done!

File: spec/railsapps/rails_60/Gemfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+source 'https://rubygems.org'
+
+gem 'rails', '~> 6.0.0.beta3'
+gem 'puma', '~> 3.11'
```
Comment (Mange) [inline spec/railsapps/rails_60/Gemfile]: The `puma` gem should not be needed, right?

File: spec/railsapps/rails_60/config/puma.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
+min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
+threads min_threads_count, max_threads_count
+port        ENV.fetch("PORT") { 3000 }
+environment ENV.fetch("RAILS_ENV") { "development" }
+plugin :tmp_restart
```
Comment (Mange) [inline spec/railsapps/rails_60/config/puma.rb]: Same with this config.

**Comment by glebm** [comment]
Done!

**Comment by Mange** [comment]
Excellent! Thank you very very much! :heart:

**Comment by Mange** [comment]
Version 2.1.0 was just released containing this.

**Comment by glebm** [comment]
Thanks!

**Comment by glebm** [comment]
I can't see it on RubyGems: https://rubygems.org/gems/roadie-rails

**Comment by Mange** [comment]
D'oh! Forgot to push the package. Will push it tomorrow when I get home.

You can use the tag on GitHub in the meantime.

Sorry about that!

Den lör 13 apr. 2019 13:20Gleb Mazovetskiy <notifications@github.com> skrev:

**Comment by Mange** [comment]
It is now pushed! Thank you for letting me know.

**Pull request merged**